README.md

```diff
@@ -42,15 +42,14 @@
 
 ### Key Features <a name="key-features"></a>
 
-- Feature 1
-- Feature 2
-- Feature 3
+- Display a list of books
+- Add a book
+- Remove a selected book
 
 ### Future Features <a name="future-features"></a>
 
-- Future Feature 1
-- Future Feature 2
-- Future Feature 3
+- Implement a DataBase
+- Deploy the Application in Render
```

Comment:
- [ ] I suggest this  Future Features section come after the Author section in your readme since you are following this [readme template](https://github.com/microverseinc/readme-template)

Reply:
Done! Thanks